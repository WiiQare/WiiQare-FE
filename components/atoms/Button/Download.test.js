import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Download from "./Download";

describe("Download button", () => {
  it("should render the component", () => {
    const { container } = render(<Download />);
    expect(container).toMatchSnapshot();
  });

  it("should render the text", () => {
    render(<Download />);
    expect(screen.getByText("Install WiiQare App")).toBeInTheDocument();
  });
});
