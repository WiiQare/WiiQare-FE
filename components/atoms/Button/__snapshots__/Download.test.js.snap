// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Download button should render the component 1`] = `
<div>
  <button
    aria-label="Install app"
    class="link-button bg-gray-800 py-2.5 text-md font-medium uppercase text-center text-white rounded-lg"
    id="setup_button"
    title="Install app"
  >
    Install WiiQare App
  </button>
</div>
`;
